roles/openshift_service_catalog/files/catalog.yaml

```diff
@@ -0,0 +1,186 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: service-catalog
+objects:
+- apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    labels:
+      app: apiserver
+    name: apiserver
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: apiserver
+    strategy:
+      rollingUpdate:
+        maxSurge: 1
+        maxUnavailable: 1
+      type: RollingUpdate
+    template:
+      metadata:
+        labels:
+          app: apiserver
+      spec:
+        containers:
+        - args:
+          - --storage-type
+          - etcd
+          - --secure-port
+          - "6443"
+          - --insecure-bind-address
+          - 0.0.0.0
+          - --insecure-port
+          - "8081"
+          - --etcd-servers
+          - http://localhost:2379
+          - -v
+          - "10"
+          - --cors-allowed-origins
+          - ${CORS_ALLOWED_ORIGIN}
+          image: quay.io/kubernetes-service-catalog/apiserver:canary
+          imagePullPolicy: Always
+          name: apiserver
+          ports:
+          - containerPort: 6443
+            protocol: TCP
+          - containerPort: 8081
+            protocol: TCP
+          resources: {}
+          terminationMessagePath: /dev/termination-log
+          volumeMounts:
+          - mountPath: /var/run/kubernetes-service-catalog
+            name: apiserver-ssl
+            readOnly: true
+        - env:
```

Comment:
If we use using the core's etcd, we do not need and should not have an etcd container in this pod.